spring-integration-core/src/main/java/org/springframework/integration/support/locks/ExpirableLockRegistry.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.integration.support.locks;
+
+/**
+ * {@link LockRegistry}s implementing this interface support removal of aged locks
```

Comment:
Doesn't fit to English: must be `LockRegistries`.
So, let's leave it in the singular! 
:smile: